src/sv-pipeline/scripts/vcf_qc/analyze_fams.R

```diff
@@ -1339,8 +1344,8 @@ if(nrow(trios)>0){
   #Read data
   trio.dat <- apply(trios[,2:4],1,function(IDs){
     IDs <- as.character(IDs)
-    return(getFamDat(dat=dat,proband=IDs[1],father=IDs[2],
-                     mother=IDs[3],biallelic=!multiallelics))
+    print(IDs)
```

Comment:
Not sure if you want to keep this `print` call in or if it was just for debugging.

Reply:
fixed